misc/multiaddr/Cargo.toml

```diff
@@ -17,6 +17,7 @@ data-encoding = "2.1"
 multihash = { package = "parity-multihash", version = "0.1.0", path = "../multihash" }
 serde = "1.0.70"
 unsigned-varint = "0.2"
+urlparse = "0.7.3"
```

Comment:
Why did you go for this crate instead of [url](https://crates.io/crates/url) which is also used in Servo?